lib/sentry_logging.rb

```diff
@@ -1,6 +1,74 @@
 # frozen_string_literal: true
 
 module SentryLogging
+  SKIP_SENTRY_EXCEPTION_TYPES = [
+    Common::Exceptions::Unauthorized,
+    Common::Exceptions::RoutingError,
+    Common::Exceptions::Forbidden,
+    Breakers::OutageException
+  ].freeze
+
+  # rubocop:disable Metrics/CyclomaticComplexity
+  # rubocop:disable Metrics/PerceivedComplexity
+  def log_error(error)
+    if skip_sentry_exception_types.include?(error.class)
+      Rails.logger.error error.message.to_s, backtrace: error.backtrace
```

Comment:
Doesn't this happen automatically in the log? This is basically "log it if we don't sentry it"

Reply:
The error passed to this method has been `rescue`d, so it wouldn't otherwise appear in the `Rails` log. Right?

Reply:
Really specific comment, but we can probably use the rails_logger method on line 15 instead of Rails.logger.error